understanding/contrast-minimum.html

```diff
@@ -459,45 +316,36 @@ <h3>Notes on formula</h3>
                </section>
                <section id="inactive-controls">
                   
-                  <h3>Inactive User Interface Components</h3>
+                  <h3>アクティブではないユーザインタフェース コンポーネント</h3>
                   
                   
-                  <p>User Interface Components that are not available for user interaction (e.g., a disabled
-                     control in HTML) are not required to meet contrast requirements. An inactive user
-                     interface component is visible but not currently operable. An example would be a submit
-                     button at the bottom of a form that is visible but cannot be activated until all the
-                     required fields in the form are completed.
+                  <p>利用者が操作に使用できないユーザインタフェース コンポーネント (例: HTML で disabled になっているコントロール) は、コントラスト要件を満たす必要はない。アクティブではないユーザインタフェース コンポーネントは、視認できてもその時点では操作ができない。例としては、フォームの下部にある送信ボタンが挙げられる。このボタンは視認できるが、フォーム内のすべての必須フィールドが入力されるまでアクティブ化できない。
```

Comment:
「視認」でいいのかどうか難しいところですが、とりあえずママにしておきますか…
```suggestion
                  <p>利用者が操作に使用できないユーザインタフェース コンポーネント (例: HTML で disabled になっているコントロール) は、コントラスト要件を満たす必要はない。アクティブではないユーザインタフェース コンポーネントは、視認できてもその時点では操作ができない。例としては、フォームの下部にある送信ボタンが挙げられる。このボタンは視認できるが、フォーム内の全ての必須フィールドが入力されるまでアクティブ化できない。
```